### ISSUE TITLE
Add support for EdDSA

### ISSUE THREAD

**Issue by kbolino** [Kitura/Swift-JWT#115]
Please add support for the `EdDSA` signature algorithm. Its use with JWT is specified by [RFC 8037](https://www.rfc-editor.org/rfc/rfc8037.html) and it would be useful for cases where one already has e.g. an Ed25519 key on hand.

**Comment by artooro** [comment]
Does this project still have an active maintainer? I wonder if someone were to contribute EdDSA support if it would even get merged.

**Comment by dannys42** [comment]
Yes, I can merge if someone is able to contribute and create appropriate unit tests.

**Comment by alinradut** [comment]
Hello @dannys42! I've sent merge request #132 which adds support for EdDSA and corresponding unit tests.

Any chance that might get reviewed and merged?

Thanks!
